src/BloomExe/web/controllers/BookSettingsApi.cs

```diff
@@ -44,7 +44,10 @@ private  void HandleBookSettings(ApiRequest request)
 					settings = DynamicJson.Parse(request.RequiredPostJson());
 					_bookSelection.CurrentSelection.TemporarilyUnlocked = settings["unlockShellBook"];
 					_pageRefreshEvent.Raise(PageRefreshEvent.SaveBehavior.SaveBeforeRefresh);
-					UpdateBookTemplateMode(settings.isTemplateBook);
+					if(((DynamicJson)settings).IsDefined("isTemplateBook"))
```

Comment:
When I commented out this feature for 3.8 in the UI, this property became unavailable